deployments/mainnet/configuration.json

```diff
@@ -0,0 +1,74 @@
+{
+  "symbol": "📈USDC",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  "baseTokenPriceFeed": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
+  "borrowMin": "1000e6",
+  "governor": "0x6d903f6003cca6255d85cca4d3b5e5146dc33925",
+  "pauseGuardian": "0xbbf3f1421d886e9b2c5d716b5192ac998af2012c",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
+    "COMP": {
+      "address": "0xc00e94cb662c3520282e6f5717214004a7f26888",
+      "priceFeed": "0xdbd020CAeF83eFd542f4De03e3cF0C28A4428bd5",
+      "decimals": "18",
+      "borrowCF": 0.65,
+      "liquidateCF": 0.7,
+      "liquidationFactor": 0.92,
```

Comment:
Have we finalized on these values yet? I can update these values to the ones in the `Comet Launch` doc if they are final. Otherwise, we can update these in a follow-up PR.

Reply:
I think we should bring it into alignment with the doc to make sure scens run, although can't promise people won't change their minds again

Reply:
Ok done, updated these values to be aligned with the doc.